site/content/docs/main/volume-group-snapshots.md

```
@@ -0,0 +1,379 @@
+- **VolumeGroupSnapshotClass (VGSClass):** A cluster-level resource that defines the configuration for a VGS, including the CSI driver and other parameters.
+- **VolumeSnapshot (VS) & VolumeSnapshotContent (VSC):** The individual volume snapshots that are created as part of the VGS process.
+
+## Velero's VGS Backup and Restore Workflow
+
+Velero's integration with VGS is designed to be as seamless as possible, automating the complexities of group snapshots. Velero supports three distinct workflows depending on your configuration:
+
+### Three VGS Workflow Branches
+
+Velero automatically selects the appropriate workflow based on your backup configuration:
+
+#### 1. VGS + Data Mover
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)